docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,811 @@
+   $make socfpga_cyclone5_sockit_arradio.dtb
+
+Building the Hardware Design
+```````````````````````````````````````````````````````````````````````````````
+
+Clone the HDL repository if it does not exist yet in local directory. Then build
+the project.
+
+.. shell::
+
+   $git clone https:github.com/analogdevicesinc/hdl.git
+   $cd hdl/projects/arradio/c5soc
+   $make
+
+After the design was built, the resulted SRAM Object File (.sof) file shall be
+converted to a Raw Binary File (.rbf).
+
+If you skipped the last section, ensure to set the architecture and cross
+compiler environment variables.
+
+.. caution::
+
+   Pay attention to directory changes to where the commands are run from,
+   and always confirm with ``pwd`` to show the current path at you terminal.
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc ; pwd
+    ~/hdl/projects/arradio/c5soc
+   $quartus_cpf -c --hps -o bitstream_compression=on \
+   $    ./arradio_c5soc.sof soc_system.rbf
+
+Building the Preloader and Bootloader Image
+```````````````````````````````````````````````````````````````````````````````
+
+This flow applies starting with release 2021_R1 / Quartus Pro version 20.1.
+For older versions of the flow see previous versions of this page on wiki
+:dokuwiki:`Altera SOC Quick Start Guide <resources/tools-software/linux-software/altera_soc_images>`.
+
+In HDL project directory, create the software/bootloader folder and clone the
+``u-boot-socfpga`` image. Before that create a new BSP settings file.
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc ; pwd
+    ~/hdl/projects/arradio/c5soc
+   $mkdir -p software/bootloader
+   $embedded_command_shell.sh bsp-create-settings --type spl \
+   $    --bsp-dir software/bootloader \
+   $    --preloader-settings-dir "hps_isw_handoff/system_bd_sys_hps" \
+   $    --settings software/bootloader/settings.bsp
+   $cd software/bootloader ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader
+   $git clone https://github.com/altera-opensource/u-boot-socfpga.git
+
+Then run the qts filter and build the preloader and bootloader images:
+
+.. shell::
+   :no-path:
+
+   $cd  ~/hdl/projects/arradio/c5soc/software/bootloader ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader
+   $cd u-boot-socfpga ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader/u-boot-socfpga
+   $./arch/arm/mach-socfpga/qts-filter.sh cyclone5 ../../../../../board/altera/cyclone5-socdk/qts/
+   $make socfpga_arria10_defconfig
+   $make
+
+Make u-boot.scr file - this file shall be copied to /BOOT partition of the SD Card
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc/software/bootloader/u-boot-socfpga ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader/u-boot-socfpga
+   $echo "load mmc 0:1 \${loadaddr} soc_system.rbf;" > u-boot.txt
+   $echo "fpga load 0 \${loadaddr} \$filesize;" >> u-boot.txt
+   $sed -i 's/ghrd_10as066n2/soc_system/g' board/altera/arria10-socdk/fit_spl_fpga.its
```

Comment:
This line is specific to Arria10, there is no ITS file for Cyclone5.

Reply:
I'll remove the line.